docs/workloads/azure/data/requirements_data_azure.md

```diff
@@ -12,28 +12,39 @@ keywords:
 ## Local development
 
 * Python 3.9+
-* Poetry https://python-poetry.org/docs/
-* (Windows users) A Linux distribution, e.g. WSL2 https://docs.microsoft.com/en-us/windows/wsl/install
+* [Poetry](https://python-poetry.org/docs/)
+* (Windows users) A Linux distribution, e.g. [WSL2](https://docs.microsoft.com/en-us/windows/wsl/install)
+* Java 8/11/17 as in the [Spark documentation](https://spark.apache.org/docs/latest/)
 
 ## Azure
 
-* Azure subscription - for deploying the solution into
-* Azure service principal - needs permissions to deploy and configure all required resources into the target subscription
-* Azure DevOps project - for running CI/CD pipelines and storing project variables
+* Azure subscription – for deploying the solution into
+* Azure service principal (Application) – needs permissions to deploy and configure all required
+resources into the target subscription
+* Azure DevOps project – for running CI/CD pipelines and storing project variables
 
 ### Azure Pipelines variable groups
 
 Our blueprint solution expects the following [variable groups](https://learn.microsoft.com/en-us/azure/devops/pipelines/library/variable-groups?view=azure-devops&tabs=yaml)
 to exist in your Azure DevOps project's Pipelines Library:
 
 * Nonprod Environment:
-  * amido-stacks-de-pipeline-nonprod
-  * amido-stacks-infra-credentials-nonprod
-  * stacks-credentials-nonprod-kv
+    * amido-stacks-de-pipeline-nonprod
+    * amido-stacks-infra-credentials-nonprod
+    * stacks-credentials-nonprod-kv
 * Prod Environment:
-  * amido-stacks-de-pipeline-prod
-  * amido-stacks-infra-credentials-prod
-  * stacks-credentials-prod-kv
+    * amido-stacks-de-pipeline-prod
+    * amido-stacks-infra-credentials-prod
+    * stacks-credentials-prod-kv
+
+These variable group names can be arbitrarily adjusted when generating new pipelines from our
+templates. Name(s) of your project's variable groups are expected to be provided in the
+configuration file, which is passed to the Datastacks CLI.
+
+Although variable group names are configurable, we still expect the specific variable names
+to exist in these groups.
+
+**TODO: list required variables in particular groups**
```

Comment:
To be updated with info from @satenderrathee

Reply:
OK yes that's true... In which case yes it makes sense to leave them in here, but I think we should have a story to update that   if possible.

Reply:
Yes, I will leave this part out when I update the variable names. Btw. during a discussion with @satenderrathee it turned out, that in our `amido-stacks-de-pipeline-env` we have some variables (e.g. credentials, and we are using them) that shouldn't really be there - we have their equivalents in the other groups, that is why we might need a story to clean that up. In the meantime I think I will just write what is the current situation. @satenderrathee should I just create a ticket to clean this up?

Reply:
I will check this and get back.

Reply:
Updated and created a story to clean that up: [6554 ADO variable groups refactor](https://amido-dev.visualstudio.com/Amido-Stacks/_workitems/edit/6554)